build/documentation/generate-typedocs.js

```diff
@@ -190,7 +190,7 @@ function addAPILinkToReadme(packageName) {
 
     var usageText = "\n" + `[API Reference](${apiLink})`;
 
-    fs.appendFile(readmePath, usageText, function(err) {
+    fs.appendFile(readmePath, usageText, function (err) {
```

Comment:
is the space after function necessary? Looks like it might not be consistently applied? Should this also apply to any function?

Reply:
this was added by prettier so it's not a manual change here - it's automated by the tool.

Reply:
Found this link to detail it: https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot